frontend/pages/ManageControlsPage/OSUpdates/components/AppleOSTargetForm/AppleOSTargetForm.tsx

```diff
@@ -178,16 +178,17 @@ const AppleOSTargetForm = ({
     }
   };
 
-  const getMinimumVersionTooltip = (platform: ApplePlatform) => {
-    switch (platform) {
-      case "darwin":
-        return "The end user sees the window until their macOS is at or above this version.";
-      case "ios":
-      case "ipados":
-        return "If the end user's host is below the minimum version, they see a notification in their Notification Center after the deadline. They can’t continue until the OS update is installed.";
-      default:
-        return "";
-    }
+  const getMinimumVersionTooltip = () => {
+    return (
+      <>
+        If an already enrolled host is below the minimum version, the
+        <br /> host is updated to exactly the minimum version.
+        <br />
+        <br /> If a new or wiped host is below the minimum version and
+        <br /> automatically enrolls (DEP), the host is updated to Apple&apos;s
```

Comment:
@noahtalerman this should be ADE

Reply:
Hey! Let's prioritize consistency and use "DEP".

At some point, we decided to use "DEP" b/c that's what we used in internal/external conversations. I think "DEP" is all over the product and docs.

More info here: https://github.com/fleetdm/fleet/pull/22128#discussion_r1763411388

Reply:
@roperzh and @ghernandez345 we're going w/ "ADE": https://github.com/fleetdm/fleet/pull/22128#discussion_r1763580543

Sorry for the back and forth